src/Umbraco.Web.BackOffice/Controllers/ContentControllerBase.cs

```diff
@@ -224,7 +224,7 @@ protected void AddCancelMessage(INotificationModel display, string message)
         {
             // if there's already a default event message, don't add our default one
             IEventMessagesFactory messages = EventMessages;
-            if (messages != null && messages.GetOrDefault().GetAll().Any(x => x.IsDefaultEventMessage))
+            if (messages?.GetOrDefault() != null && messages.GetOrDefault().GetAll().Any(x => x.IsDefaultEventMessage))
```

Comment:
This is now calling `GetOrDefault()` twice, which can be avoided by using pattern matching:
```suggestion
            if (messages?.GetOrDefault() is EventMessages eventMessages &&
                eventMessages.GetAll().Any(x => x.IsDefaultEventMessage) == true)
```

Reply:
Or using multiple null-conditional operators:
```suggestion
            if (messages?.GetOrDefault()?.GetAll().Any(x => x.IsDefaultEventMessage) == true)
```